src/Console/Commands/CheckCommand.php

```diff
@@ -272,6 +274,8 @@ private function renderLineLessIssue(Issue $issue): void
             $issue,
         );
 
+        $suggestionsHelperText = ($suggestions > '') ? 'Did you mean:' : 'No suggestions found.';
```

Comment:
See previous comment ^^
```suggestion
        $suggestionsHelperText = ($suggestions !=== '') ? 'Did you mean:' : 'No suggestions found.';
```